webhook-update.js

```diff
@@ -0,0 +1,53 @@
+
+/* update existing hooks */
+const doAsync = require("doasync");
+var GH = require("./gh");
+var config = require("./config.json");
+var hookURL = config.hookURL;
+
+
+if (require.main === module) {
+    var Store = require("./store");
+    config.logToConsole = false;
+    const store = new Store(config);
+    store.repos(async (err, data) => {
```

Comment:
not critical, but would be nice to use `doAsync` and `await` instead of callbacks

Reply:
addressed in https://github.com/w3c/ash-nazg/pull/224/commits/2d70617d87ca1aa4f0b68f01bbab15039ddcbd2a